bin/crew

```diff
@@ -824,8 +824,8 @@ def unpack(meta)
       if !File.exist?("#{CREW_PREFIX}/bin/tar") and %x[tar --version | head -n 1 | awk '{print $4}'] < '1.31'
         abort 'A newer version of tar may be needed for this install. Please install it first with \'crew install tar\''.lightred
       end
-      if !File.exist?("#{CREW_PREFIX}/bin/zstd")
-        abort 'zstd is needed for this install. Please install it first with \'crew install zstd\''.lightred
+      if !File.exist?("#{CREW_PREFIX}/bin/zstd") and !File.exist?("#{CREW_MUSL_PREFIX}/bin/zstd")
+        abort 'zstd is needed for this install. Please (re)install it first with \'crew remove musl_zstd zstd ; crew install musl_zstd ; crew install zstd\''.lightred
```

Comment:
Could this be replaced with `crew reinstall musl_zstd zstd`?

Reply:
So the answer is no. `musl_zstd` needs to be installed separately first. And I think it is cleaned to uninstall first, since if rsync fails, crew still records the package as installed.

Reply:
rsync needs `zstd` installed.
tar needs to have `musl_zstd` and/or `zstd` to be able to process `zst` packages.